README.md

```diff
@@ -112,6 +112,88 @@ If the `current-context` is set to the name of one of the clusters then the
 `--context` argument may be omitted, and the cluster named by `current-context`
 will be used.
 
+## Kubernetes (example)
+Kuberos can be ran inside a cluster as long as it can still communicate with your OIDC provider from inside the pod and your OIDC provider is set to redirect to your Kuberos endpoint (NodePort, LoadBalancer, etc).
+
+The configuration below is meant to serve as a template and **not** something that is plug-and-play. You will need to adjust your DNS / nameserver helpers, Dex information, and optionally how you ingress your traffic.
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: kuberos
+  name: kuberos
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: kuberos
+    spec:
+      hostAliases:                  #
+      - ip: "192.168.1.1"           # OPTIONAL TO HELP ROUTE TRAFFIC TO DEX / OIDC
+        hostnames:                  #
+        - "dex.oidc.example.com"    # OPTIONAL TO HELP ROUTE TRAFFIC TO DEX / OIDC
+      containers:
+      - image: negz/kuberos:latest
+        name: kuberos
+        command: ["/kuberos", "https://dex.oidc.example.com", "example-app", "/cfg/secret", "/cfg/template"]
+        ports:
+        - name: http
+          containerPort: 10003      # DEFAULT FOR KUBEROS 10003, MATCHES NODEPORT SERVICE
+        volumeMounts:
+        - name: config
+          mountPath: /cfg
+      dnsConfig:
+        nameservers:
+          - 8.8.8.8                 # USING GOOGLE DNS FOR LOOKUPS
```

Comment:
Is this something that commonly needs to be set? I don't think I've ever explicitly configured my pods' nameserver, but rather have let them use KubeDNS.

Reply:
I don't think it is common, but I ran into some funky things using an air-gapped k8s cluster. Ok with removing this from the example.

Reply:
Sounds good - I agree there's value in the `hostAliases`.